### DISCUSSION TITLE
RoomGPT - Replicate: The requested resource could not be found

### DISCUSSION THREAD

**Discussion by bkoubik88** [vercel/next.js#49502] [Help]
### Summary

Hello everyone. I'm trying to recreate the RommGPT app: https://vercel.com/templates/next.js/room-GPT.
When calling the POST request, however, I get a 404 error: 'The requested resource could not be found.' I've already tried it with Postman, same problem. I also see in the debugger: POST https://api.replicate.com /v1/predictions net::ERR_FAILED
dispat and a CORS policy error. The app is hosted in VERCEL.

![image](https://user-images.githubusercontent.com/29425269/237008878-ad93cb2a-6620-470b-a46a-92f6b68113a6.png)

` let startResponse = await fetch(
      "https://api.replicate.com/v1/predictions",
      {
        method: "POST",
        headers: {
          "Content-Type": "application/json",
          Authorization: "Token " + process.env.REPLICATE_API_KEY,
        },
        body: JSON.stringify({
          version:
            "8ebda4c70b3ea2a2bf86e44595afb562a2cdf85525c620f1671a78113c9f325b",
          input: {
            image: "base64...",
            structure: "hough",
            prompt: "a modern office room",
            scale: 9,
            a_prompt:
              "best quality, photo from Pinterest, interior, cinematic photo, ultra-detailed, ultra-realistic, award-winning, interior design, natural lighting",
            n_prompt:
              "longbody, lowres, bad anatomy, bad hands, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality",
          },
        }),
      }
    );`

**Comment by icyJoseph** [comment]
Are you trying to reach for api.replicate from the browser? I think that one is only to be reaches from the server side.
